app/test/obituary_test.rb

```diff
@@ -33,4 +33,14 @@ def test_it_returns_formatted_obituary_data_by_date
 			get '/date', beginning_date: '20170101', ending_date: '20190101'
 			assert_equal "obituary", JSON.parse(last_response.body).first['data']['type']
 	end
+
+	def test_returns_most_recent_obituaries
```

Comment:
Will build out these test to better reflect what is returned, as discussed in meeting, on another branch

Reply:
Sounds good, yeah it definitely makes sense to start testing edge cases and sad paths here.